Bio/PDB/PDBParser.py

```diff
@@ -142,8 +142,14 @@ def _get_header(self, header_coords_trailer):
     def _parse_coordinates(self, coords_trailer):
         """Parse the atomic data in the PDB file (PRIVATE)."""
         allowed_records = {
-            "ATOM  ", "HETATM", "MODEL ", "ENDMDL", "TER   ", "ANISOU",
-            "SIGATM", "SIGUIJ",  # older 2.3 format specs
+            "ATOM  ",
+            "HETATM",
+            "MODEL ",
+            "ENDMDL",
+            "TER   ",
+            "ANISOU",
+            "SIGATM",
+            "SIGUIJ",  # older 2.3 format specs
```

Comment:
Argh. I disagree with black sometimes... I'd move this comment to the line above them, since both SIGATM and SIGUIJ are from the previous format spec.

Reply:
Comment placement is something best tweaked after running black. Sorry, my oversight in the recent TravisCI changes let this slip by (``black --diff`` only shows the differences but does not report an error, thus the change here to ``black --diff --check`` instead).

Reply:
Is this OK now (moved the comments)?